.envs/.local/.caluma

```diff
@@ -5,6 +5,7 @@ OIDC_VERIFY_SSL=False
 OIDC_USERINFO_ENDPOINT=https://mysagw.local/auth/realms/mysagw/protocol/openid-connect/userinfo
 # Multiple ADMINS need to be comma separated
 ADMINS=Test Example <test@example.com>
+DISABLE_INTROSPECTION=true
```

Comment:
Caluma 10.1.1 (referenced in mysagw) still has a bug that prevents `__typename`, which the frontend absolutely must have.

Before setting `DISABLE_INTROSPECTION=true`, we'll need another Caluma release that contains this bugfix:

https://github.com/projectcaluma/caluma/pull/2155

Reply:
Lol, I know, I've just confused `true` and `false` :see_no_evil:  It's fixed now.

Reply:
Happens to the best of us :rofl: